css/style.css

```diff
@@ -0,0 +1,248 @@
+body{
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    color: var(--color-white);
+    margin: 0;
+    display: block;
+
+}
+
+.container{
+    
+    background-color: #262647;
+    margin-top: 3px;
+    margin-bottom: 3px;
+    margin-right: 300px;
+    margin-left: 300px;
+    padding: 16px;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+:root{
+    --color-white: #f3f3f3;
+    --color-darkblue: #1b1b32;
+    --color-darkblue-alpha: rgba(27,27,50,0.8);
+    --color-green: #37af65;
+}
+
+body::before{
+    content: '';
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: -1;
+    background: var(--color-darkblue);
+    background-image: linear-gradient(115deg, rgba(58,58,158,0.8), rgba(136,136,206,0.7)), 
+    url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+*::before{
+    box-sizing: border-box;
+}
+
+.header{
+    padding: 0 0.625rem;
+    margin-bottom: 1.875rem;
+    display: block;
+}
+
+.text-center{
+    text-align: center;
+    
+}
+
+h1{
+    margin-top: 0;
+    margin-bottom: 0.5rem;
+    font-weight: 400;
+    line-height: 1.5rem;
+    display: block;
+    font-size: 2em;
+    margin-block-start: 0.67em;
+    margin-block-end: 00.67em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+}
+
+.discripton{
+    font-style: italic;
+    font-weight: 200;
+    text-shadow: 1px 1px 1px rgb(0, 0,0 / 40%);
+
+
+}
```

Comment:
Remove unnecessary space after the end of the style. This is makes your code not clean and un easy to read by other programmers.
Its not a CSS best practice

Reply:
> Remove unnecessary space after the end of the style. This is makes your code not clean and un easy to read by other programmers. Its not a CSS best practice

- [ ] You must have wanted to say `this makes` not `this is makes`. Also, I would recommend your comment to sound friendly and advisory. It would sound good than having a command tone. What do you think about changing your first sentence which is "Remove unnecessary space after the end of the style." to "What do you think about removing the spaces after the end of the style? Since the spaces aren't doing anything. This will even make it professional and organized."

### Consider applying this comment to all your comments.